website/README.md

```diff
@@ -118,6 +118,42 @@ The code for the client is located under `src/client/`. The webpack build create
 
 Scripts related to deployment, like the Dockerfile, are under `deploy`. Note: even though the scripts are under `deploy`, you must run them from the **root of the repository**; they are sensitive to `cwd`.
 
+## Query Parameters
+
+You can pass certain parameters in the Snack URL to customize the behavior of the Snack.
+
+Here is a summary of all the parameter options that you can use:
+
+| parameter | default | description | type |
+| -------- | ------- | ----------- | ------ |
+| **dependencies** | | Dependencies to be used by your Snack beyond common dependencies and the ones added in the Snack's `package.json`. [Read more about Snack dependencies](_`string[https://github.com/expo/snack/blob/main/docs/snack-sdk.md#using-dependencies]`_). | [SnackDependency](https://github.com/expo/snack/blob/main/packages/snack-content/src/types.ts#L64) |
```

Comment:
For the dependencies, I think it would be easier to just mention:
> Comma separated list of `<name>@<version>`

That should be accurate of what we expect here instead of a URL encoded object.